packages/react/src/components/headerUniversalBar/HeaderUniversalBar.module.scss

```diff
@@ -0,0 +1,53 @@
+@value x-small-down, small-down from "../../styles/breakpoints.css";
+
+.headerUniversalBar {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  min-height: 48px;
+  background: var(--color-black-5);
+  padding: 0 var(--spacing-m);
+}
+
+.headerUniversalBarList {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.universalBarMainLinkContainer {
+  display: flex;
+  justify-self: start;
+  flex: 1;
+}
+
+.universalBarSecondaryLinkContainer {
+  display: flex;
+  justify-self: end;
+  margin-right: var(--spacing-l);
+}
+
+.universalBarSecondaryLinkContainer:last-child {
+  margin-right: 0;
+}
+
+.universalBarLink.universalBarLink {
+  font-size: var(--fontsize-body-m);
+}
+
+/* MEDIA */
+@media small-down {
+  .universalBarSecondaryLinkContainer {
+    margin-right: var(--spacing-m);
```

Comment:
Should we start using flex's gap property instead of margins for item spacing?

Reply:
Updated the demo, test it out.